### PR TITLE
New Draft of charter updates

### PR DESCRIPTION
**Pull request by lmbr-pip**
This is based on https://github.com/o3de/sig-network/pull/15 with some minor changes.

Original PR has DCO issue so closing out that PR in favor of this one so it can be submitted cleanly.

Only substantive change is to add additional Chair responsibilities (taken from https://github.com/o3de/sig-testing/blob/main/governance/SIG%20Testing%20Charter.md) 

Signed-off-by: Pip Potter <61438964+lmbr-pip@users.noreply.github.com>

### CODE REVIEW

File: governance/SIG Network Charter.md [hunk 1]
```diff
@@ -1,97 +1,109 @@
...
 
 ###Network:
-- Design and maintain client and server protocol communication 
+- Design and maintain client and server protocol communication.
 - Define, publish, and maintain network protocol and data packet design model.
 - Responsible for network presence and state replication.
-- Create and Maintain network simulation tools for testing network condititons.
-
-- Design and maintain multiplayer component architecture supporting client hosted and dedicated server strategies
+- Create and Maintain network simulation tools for testing network conditions.
```
Comment (AMZN-Olex) [inline governance/SIG Network Charter.md:36]: loser case? Maintain->maintain

**Review by AMZN-Olex — APPROVED** [review]

File: governance/SIG Network Charter.md [hunk 2]
```diff
@@ -1,97 +1,109 @@
...
-- Define and maintain entity replication network model
-- Responsible for abstraction of console design for dedicated / headless server implementation (core / init)
+- Define, standardize, and maintain interfaces for features based on cloud services.
+- Define and maintain code to enable cloud data services communication.
+
+###Cross-cutting Processes
+- Define client and server build targets for supported platforms, cross-cutting issue with `#sig-core` and `#sig-build`.
+- Define communication protocol and stacks.
+- Define and maintain entity replication network model.
+- Responsible for abstraction of console design for dedicated / headless server implementation (core / init).
```
Comment (AMZN-Gene) [inline governance/SIG Network Charter.md:49]: what's core/init? the base-classes and patterns used for the core functionality and how systems are initialized?

File: governance/SIG Network Charter.md [hunk 3]
```diff
@@ -1,97 +1,109 @@
...
+     - Issues: https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Asig%2Fnetwork
+     - PRs: https://github.com/o3de/o3de/pulls?q=is%3Apr+is%3Aopen+label%3Asig%2Fnetwork
+- Meeting agenda & Notes: 
+     - Agendas: https://github.com/o3de/sig-network/labels/mtg-agenda
+     - Meeting Notes: https://github.com/o3de/sig-network/tree/main/meetings 
 
 ## Roles and Organization Management
 
-SIG Docs adheres to the standards for roles and organization management as specified by <sig-governance>. This SIG opts in to updates and modifications to <sig-governance>
+SIG-network adheres to the standards for roles and organization management as specified by [O3DE Foundation](https://github.com/o3de/community). This SIG opts in to updates and modifications to This SIG opts in to updates and modifications to the Foundation's guidance (referred to below as <sig-governance>)
```
Comment (AMZN-Gene) [inline governance/SIG Network Charter.md:82]: duplicate? "This SIG opts in to updates and modifications to" is written 2x

**Review by AMZN-Gene — APPROVED** [review]

File: governance/SIG Network Charter.md (hunk 3 above)
Comment (lmbr-pip) [reply governance/SIG Network Charter.md:82]: Opps, will fix. Thanks.

File: governance/SIG Network Charter.md (hunk 1 above)
Comment (lmbr-pip) [reply governance/SIG Network Charter.md:36]: Will fix thanks.

**Pull request merged**